CharlieBackend.Api.UnitTes/EventOcurranceStorageParserTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using CharlieBackend.Business.Services.ScheduleServiceFolder;
+using CharlieBackend.Core.DTO.Schedule;
+using CharlieBackend.Core.Entities;
+using System;
+using System.Collections.Generic;
+using Xunit;
+using FluentAssertions;
+
+namespace CharlieBackend.Api.UnitTest
+{
+    public class EventOcurranceStorageParserTests
+    {
+        [Fact]
+        public void GetPatternStorageValue_ValidPattern_ShouldReturnSuccessResult()
```

Comment:
Consider to rename to ShouldReturnExpectedModel

Reply:
Ok, test method renamed